README.md

```diff
@@ -553,8 +549,17 @@ As you can see, the balenaFin is attached to drive `/dev/disk4`. We can now use
 another balena CLI command to write the image to that drive, which will
 provision the device:
 ```
-$ balena os initialize balena-fin-image.img --type fincm3 --drive /dev/disk3 --yes
-...
+$ balena os initialize balena-fin-image.img --type fincm3 --drive /dev/disk4 --yes
+Initializing device
+
+Note: Initializing the device may ask for administrative permissions
+because we need to access the raw devices directly.
+Going to erase /dev/disk4.
+Admin privileges required: you may be asked for your computer password to continue.
+Password:
+Writing Device OS [========================] 100% eta 0s
+Validating Device OS [========================] 100% eta 0s
+You can safely remove /dev/disk4 now
 ```
 Note that you *must* supply the device type and the drive to use to write to.
```

Comment:
Perhaps this changed with the recent fix / PR, but I've found that it is **not** necessary to specify the drive as a command-line argument. I've tested on macOS, Linux and Windows. The CLI will offer a list of drives to choose from:
```
$ balena os initialize test.img --type raspberrypi3
Initializing device

Note: Initializing the device may ask for administrative permissions
because we need to access the raw devices directly.
? Select a drive /dev/sda (8.0 GB) - TrekStor USB Stick QU
? This will erase /dev/sda. Are you sure? Yes
Admin privileges required: you may be asked for your computer password to continue.
Writing Device OS [========================] 100% eta 0s
Validating Device OS [========================] 100% eta 0s
You can safely remove /dev/sda now
```

By the way, excellent work writing the CLI masterclasses! 💯

Reply:
@pdcastro You've spotted something I meant to write and didn't, that the drive option lets us write non-interactively. I'll add that pre-merge! And thank you for the kind comment (and for fixing the issues)! :)